iceoryx_posh/include/iceoryx_posh/version/compatibility_check_level.hpp

```diff
@@ -28,6 +30,32 @@ enum class CompatibilityCheckLevel
     BUILD_DATE
 };
 
+inline iox::log::LogStream& operator<<(iox::log::LogStream& logstream, const version::CompatibilityCheckLevel& level)
```

Comment:
This streaming operator is not unit tested according to codecov. Could you please add some unit tests.

Reply:
oh, that's will be interesting. I guess you have to look at `test_logstream` ... maybe the `Logger_Mock` should be moved to the testutils

Reply:
Done